shared/hack/install.sh

```diff
@@ -134,9 +134,8 @@ install_argocd() {
 }
 
 install_bitwarden() {
-    curl "${CURL_OPTS[@]}" -o "$TMPBIN/bw.zip" "https://github.com/bitwarden/clients/releases/download/cli-${BITWARDEN_VERSION}/bw-linux-${BITWARDEN_VERSION:1}.zip"
-    unzip "$TMPBIN/bw.zip" -d "$TMPBIN/"
-    rm -rf "$TMPBIN/bw.zip"
```

Comment:
Removing this line will copy the zip file into BIN_DIR as move_bin is called after this. We don't want the zip file, just the executable.